sideboard/configspec.ini

```diff
@@ -52,18 +60,25 @@ checker.check_skipped_app_config = boolean(default=False)
 [rpc_services]
 ___many___ = string
 
-#[[___many___]]
-#jsonrpc_only = boolean(default=False)
+[[__many__]]
+jsonrpc_only = boolean(default=False)
 
 
 [loggers]
-root = option("DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
-cherrypy.error = option("DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
-cherrypy.access = option("DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
-__many__ = option("DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
+root = option("TRACE", "DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="DEBUG")
+cherrypy.error = option("TRACE", "DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="DEBUG")
+cherrypy.access = option("TRACE", "DEBUG", "INFO", "WARNING", "WARN", "ERROR", "CRITICAL", default="CRITICAL")
+__many__ = option("TRACE", "DEBUG", "INFO", "WARN", "WARNING", "ERROR", "CRITICAL", default="INFO")
 
 [handlers]
 [[__many__]]
 formatter = string(default="default")
 ___many___ = string()
 
+[formatters]
+[[default]]
+format = string(default="$$(asctime)s [$$(levelname)s] $$(name)s: $$(message)s")
+datefmt = string(default="")
+[[__many__]]
+format = string
+datefmt = string(default="")
```

Comment:
Sideboard now supports custom format strings instead of hard-coding its log message format.